conformance_tests/tools/metrics/src/test_metric.cpp

```diff
@@ -824,10 +827,12 @@ void run_multi_device_query_load_test(
 
   auto metric_group_info_0 = lzt::get_metric_group_info(
       device_0, ZET_METRIC_GROUP_SAMPLING_TYPE_FLAG_EVENT_BASED, true);
-  ASSERT_GT(metric_group_info_0.size(), 0u) << "No query metric groups found";
+  ASSERT_GT(metric_group_info_0.size(), 0u)
```

Comment:
this is a grey area: supporting query is a HW feature (HW must support command list generated ). But all platforms support it.

Reply:
Not updating at the moment, thank you.